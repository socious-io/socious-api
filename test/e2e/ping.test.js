import supertest from 'supertest'
import { app } from '../../src/index.js'

let server, request

beforeAll(() => {
  server = app.listen()
  request = supertest(server)
})

test('ping works', async () => {
  const response = await request.get('/ping')
  expect(response.status).toBe(200)
  expect(response.text).toMatchSnapshot()
})

afterAll((done) => {
  server.close(done)
})
