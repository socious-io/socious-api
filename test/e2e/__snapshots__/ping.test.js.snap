// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ping works 1`] = `"pong"`;
